README.md

```diff
@@ -30,3 +31,30 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+
+
+**MY ANSWER:**
+
+The recurrence relation can be expressed as $T(n) = 3T(n/3) + n^5$ as the function makes 3 recursive calls of size n/3, and there are $n^5$ loop iterations.
+
+We can use the base case T(1) = $O$(1) to solve our recurrence relation.
```

Comment:
When does the base case happen and how much work do you do then? Not asymptotically.